tests/helpers/iframes.js

```diff
@@ -0,0 +1,7 @@
+// This file is a helper to abstract the retrieval of the iframes on the paywall,
+// so that any re-ordering of frames is controlled in a single location
+
+module.exports = {
```

Comment:
Are the order here and in the subsequent product code important? What happens if the order changes?

Reply:
yes, prior to this PR, the grabbing of which iframe was which was done in the test. When I needed to change the ordering, it upset the frame number. As it turns out, I can't reliably identify frames by their source in puppeteer, although I did not spend a long time because there were other pressing things. If this becomes an issue again, we should find another way to identify the frames.

What happens is the integration tests time out on finding any of the things, because they are looking in the wrong iframe